scripts/format-stale-pr-message.js

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env node
+
+/**
+ * Format stale PR data into a Slack message payload
+ *
+ * This script reads PR data from /tmp/stale-prs.json and generates
+ * a formatted Slack message at /tmp/slack-message.json
+ */
+
+const fs = require( 'fs' );
+
+// Read stale PRs data
+const stalePRs = JSON.parse( fs.readFileSync( '/tmp/stale-prs.json', 'utf8' ) );
+
+if ( stalePRs.length === 0 ) {
+	console.log( 'No stale PRs to format' );
+	process.exit( 0 );
+}
+
+// Calculate how long ago each PR was updated
+function getTimeAgo( updatedAt ) {
+	const now = Date.now();
+	const updated = new Date( updatedAt ).getTime();
+	const diffMs = now - updated;
+	const diffHours = Math.floor( diffMs / ( 1000 * 60 * 60 ) );
+	const diffDays = Math.floor( diffHours / 24 );
+
+	if ( diffDays > 0 ) {
+		return `${ diffDays } day${ diffDays > 1 ? 's' : '' } ago`;
+	} else {
+		return `${ diffHours } hour${ diffHours > 1 ? 's' : '' } ago`;
+	}
+}
+
+// Format PR status
+function getStatusEmoji( pr ) {
+	if ( pr.reviewDecision === 'REVIEW_REQUIRED' ) {
+		return '▶️';
+	} else if ( pr.reviewDecision === 'CHANGES_REQUESTED' ) {
+		return '⏩️️';
+	} else {
+		return '⏸️';
```

Comment:
Have we considered using custom slack emojis?

```suggestion
		return ':pr-open:';
	} else if ( pr.reviewDecision === 'CHANGES_REQUESTED' ) {
		return ':github-changes-requested:';
	} else {
		return '⏸️';
```
For example: 

<img width="89" height="85" alt="Image" src="https://github.com/user-attachments/assets/4d1d2a79-8d4b-49e6-a019-b12ddfa524c8" />
<img width="183" height="85" alt="Image" src="https://github.com/user-attachments/assets/4bbde02a-3305-4709-992a-42259fefbec0" />

Reply:
I tried using red/yellow/green icons, but the message seemed too aggressive.

Reply:
I think the current emojis are fine 👍 I like that they are not too forward which makes you feel like it is a normal part of your workflow instead of something urgent